File: VyKundeservice/ClientApp/src/app/models/user-question.ts
import { Category } from './category';

export class UserQuestionDTO {
    id: number;
    firstname: string;
    lastname: string;
    question: string;
    categoryId: number;
}

export class UserQuestion {
    id: number;
    firstname: string;
    lastname: string;
    question: string;
    categoryId: number;
    category: Category;
}

File: VyKundeservice/Models/UserQuestion.cs
namespace VyKundeservice.Models 
{
    public class UserQuestion
    {
        public int Id { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public string Question { get; set; }
        public int CategoryId { get; set; }
        public Category Category { get; set; }
    }
}